content/hi/docs/reference/glossary/etcd.md

```diff
@@ -0,0 +1,20 @@
+---
+title: etcd
+id: etcd
+date: 2018-04-12
+full_link: /docs/tasks/administer-cluster/configure-upgrade-etcd/
+short_description: >
+   सभी क्लस्टर डेटा के लिए कुबेरनेट्स के बैकिंग स्टोर के रूप में उपयोग किया जाने वाला सुसंगत और अत्यधिक उपलब्ध की-वैल्यू स्टोर।
```

Comment:
उच्च उपलब्धता seems more correct

Reply:
@kundan2707 How would you suggest the sentence be rephrased then?

Reply:
@divya-mohan0209 
सभी क्लस्टर डेटा के लिए कुबेरनेट्स के बैकिंग स्टोर के रूप में उपयोग किया जाने वाला सुसंगत एवं उच्च उपलब्धता वाला  की-वैल्यू स्टोर

Reply:
@amolmote : @kundan2707 does seem to be correct here since अत्यधिक = Excessive & उच्च = High. Will you please be able to make the changes whenever you're free?

Reply:
hi @divya-mohan0209, done i have made changes